libs/feature/map/src/lib/utils/map-utils-wms.service.ts

```diff
@@ -24,17 +28,35 @@ export class MapUtilsWMSService {
 
   getLayerLonLatBBox(layer: MapContextLayerWmsModel) {
     return this.getLayerFull(layer).pipe(
-      map((wmsLayerFull) => this.getLonLatBBox(wmsLayerFull))
+      switchMap((wmsLayerFull) => from(this.getLonLatBBox(wmsLayerFull)))
     )
   }
 
-  getLonLatBBox(wmsLayerFull: WmsLayerFull) {
+  async getProjFromEPSG(EPSGCode) {
+    return fromEPSGCode(EPSGCode)
+  }
```

Comment:
Is this function really needed? Calling `fromEPSGCode` directly elsewhere should work